docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -69,7 +71,31 @@ Installing YugabyteDB Anywhere on Airgapped hosts, without access to any Interne
 - Having YugabyteDB Anywhere airgapped install package. Contact Yugabyte Support for more information.
 - Signing the Yugabyte license agreement. Contact Yugabyte Support for more information.
 
-### Pull and push YugabyteDB Docker images to private container registry
+### Kubernetes-based installations
+
+A Kubernetes-based installation of YugabyteDB Anywhere requires you to address concerns related to security and core dump collection.
+
+#### Specify ulimit and remember the location of core dumps
+
+The core dump collection in Kubernetes requires special care due to the fact that `core_pattern` is not isolated in cgroup drivers.
+
+You need to ensure that core dumps are enabled on the underlying Kubernetes node. Running the `ulimit -c` command within a Kubernetes pod or node must produce a large non-zero value or the `unlimited` value as an output. For more information, see [How to enable core dumps](https://www.ibm.com/support/pages/how-do-i-enable-core-dumps). 
+
+To be able to locate your core dumps, you should be aware of the fact that the location to which core dumps are written depends on the sysctl `kernel.core_pattern` setting. For more information, see [Linux manual: core(5)](https://man7.org/linux/man-pages/man5/core.5.html#:~:text=Naming of core dump files).
```

Comment:
The Linux manual link is broken in the preview.

Reply:
Yes, I saw it and fixed in the updated files